anyway/widgets/suburban_widgets/accident_count_by_car_type_widget.py

```diff
@@ -22,8 +26,10 @@ def __init__(self, request_params: RequestParams):
         self.rank = 17
 
     def generate_items(self) -> None:
-        self.items = AccidentCountByCarTypeWidget.get_stats_accidents_by_car_type_with_national_data(
-            self.request_params
+        self.items = (

```

Comment:
why this change? please revert if it's not needed. in general accidental moot changes make it more difficult to perform CRs